kaniko_test.go

```diff
@@ -280,3 +284,230 @@ func TestTarPathValidation(t *testing.T) {
 		})
 	}
 }
+
+func TestLocalTarballPush(t *testing.T) {
+	tests := []struct {
+		name          string
+		localTarPath  string
```

Comment:
```suggestion
		sourceTarPath  string
```
update all the refs to sourceTarPath

Reply:
Updated. ✅